build-tools/plots-to-appinsights/AppInsights.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using System;
+using System.Collections.Generic;
+
+namespace ProcessPlotCSVFile
+{
+	internal class AppInsights
+	{
+		private TelemetryClient TelemetryClient { get; set; } = null;
```

Comment:
No need for this to be a *property* when it's `private`; leave it a field.

If the property weren't `private`, no need for it to have a setter either, or a non-private setter, and no need for `= null` either.

Reply:
Indeed the null/empty checks aren't needed as the constructor should keep everything honest.
Addressed with commit: 135324e3afd3da9c3c91a9de110ae83e56df3353